src/cmd.rs

```diff
@@ -0,0 +1,52 @@
+use std::path::Path;
+use std::{env, io};
+
+/// Read command and args.
```

Comment:
What is the meaning of the triple slash comment style?

Reply:
javadoc like